server/src/services/group_permissions.service.ts

```
@@ -27,13 +27,19 @@ export class GroupPermissionsService {
     private appRepository: Repository<App>,
 
     private usersService: UsersService
-  ) {}
+  ) { }
 
   async create(user: User, group: string): Promise<GroupPermission> {
     if (!group || group === '') {
       throw new BadRequestException('Cannot create group without name');
     }
 
+    const reservedGroups = ['All Users', 'all_users', 'Admin', 'admin'];
+
+    if (reservedGroups.includes(group)) {
+      throw new BadRequestException('Cannot create group with system reserved group name');
```

shall we throw the same error
```
Group name already exist
```

---

Sure, Will do

---

Done